apps/api/src/domain/models/bibliographic-reference/interfaces/bibliographic-reference-data.interface.ts

```diff
@@ -1,5 +1,6 @@
+import BaseDomainModel from '../../BaseDomainModel';
 import { BibliographicReferenceType } from '../types/BibliographicReferenceType';
 
-export interface IBibliographicReferenceData {
+export interface IBibliographicReferenceData extends BaseDomainModel {
```

Comment:
Same for spatial feature?

Reply:
No. We have decided to opt out of OO baggage and use a plain-old object for the `geometry`. So ISpatialFeature should extend Resource (this was the case already), but IGeometricFeature should **not** extend `BaseDomainModel`.